connectivity/connectivity-demos-test/src_test/rest/bpm/TestAsyncInvocation.java

```diff
@@ -22,12 +30,17 @@ void callBatchAsync(BpmClient bpmClient) throws InterruptedException
     ExecutionResult result = bpmClient.start().process("rest/asyncInvocation/callBatchAsync.ivp").execute();
     assertThat(result).isNotNull();
     
-    Thread.sleep(4000);
```

Comment:
get rid of this bad practice; to have strict timeouts ...  the process is being delayed by design for 3 secs; but we can't assume how fast the firing of the signal will be available ... Q.E.D in our frequently failing pipeline in the night. :watch: